### ISSUE TITLE
'file.exclude' Not Working

### ISSUE THREAD

**Issue by AntumDeluge** [lunarmodules/LDoc#279]
I have tried using the *exclude* table from within *config.ld* to ignore a directory:
```lua
file = {
	'init.lua',
	'api.lua',
	exclude = {'engine'},
}
```
But it continues to parse the files within:
> .../engine/mobf/api.lua:30: ?: 'class' cannot have multiple values; {function,module}

I tried explicitly using the file name with no success:
```lua
file = {
	'init.lua',
	'api.lua',
	exclude = {'engine/mobf/api.lua'},
}
```
Also, *file* itself does not seem to be working, which is probably why *file.exclude* doesn't. The only working options seems to be to append "./" or a single filename to the command line.
I am using LDoc 1.4.6 at commit f91c318.

**Comment by flyingdutchman23** [comment]
For people with a similar problem: 
For me the file configuration without the exclude worked with this command `ldoc.lua .` and the following confing:
```
project="bla"
add_language_extension("lc", "c")
file = {
    './some/path',
    './another/longer/path/,
}
```

I used the self-compiled ldoc at commit `f91c318`

**Comment by AntumDeluge** [comment]
It turns out that my issue is that LDoc only recognizes `file` from the config if the `file` command line parameter is `.`. I created a pull request to fix this: #356
